opentelemetry-sdk/src/opentelemetry/sdk/configuration/__init__.py

```diff
@@ -33,7 +33,7 @@
 EXPORTER_OTLP = "otlp"
 EXPORTER_OTLP_SPAN = "otlp_span"
 EXPORTER_OTLP_METRIC = "otlp_metric"
-_DEFAULT_EXPORTER = EXPORTER_OTLP
+_DEFAULT_EXPORTER = "none"
```

Comment:
Do we still need this?

Reply:
this is changing the default in the SDK configurator *not* to make a decision for users as to what exporter to configure

Reply:
Why not remove the line completely?

Reply:
Removed.